### ISSUE TITLE
[Request] Change "Only download over Wi-Fi" to "Don't download over metered connections"

### ISSUE THREAD

**Issue by KarolS** [inorichi/tachiyomi#1944]
**App version:** 0.8.2

**Android version:** 6.0

**Issue/Request:**

The option "Only download over Wi-Fi" should be renamed to "Don't download over metered connections" and the error message "No wifi connection available" should be renamed to something else too.

**Steps to reproduce**

 1. Set a network as metered (for example by going to Settings – Data Usage – Wi-Fi – ⋮ – Mobile hotspots)
 2. Forget you did it
 3. Connect one day to that metered wi-fi network 
 4. Tell Tachiyomi to download some chapters
 5. Get a "No wifi connection available" error
 6. Get very confused, as wi-fi clearly works

**Comment by AmineI** [comment]
I find that less clear for the user. As an unlimited 4G connection wouldn't necessarily be called metered. Maybe "download only on unmetered WiFi connections"

**Comment by KarolS** [comment]
Tachiyomi does not check if your connection is cellural, wi-fi or [wired](https://plugable.com/wp-content/uploads/2015/01/2_5121.jpg), it only checks metered/unmetered.

The code uses `ConnectivityManager#isActiveNetworkMetered()`, which is the recommended way of checking if downloading large amounts of data should be allowed.
If the system knows that your cellular network is unmetered, then it will report so to Tachiyomi and downloading will proceed.

**Comment by CarlosEsco** [comment]
Probably would rephrase it to:

Don't download over metered data connections

**Comment by AmineI** [comment]
> Tachiyomi does not check if your connection is cellural, wi-fi or [wired](https://plugable.com/wp-content/uploads/2015/01/2_5121.jpg), it only checks metered/unmetered.
> 
> The code uses `ConnectivityManager#isActiveNetworkMetered()`, which is the recommended way of checking if downloading large amounts of data should be allowed.
> If the system knows that your cellular network is unmetered, then it will report so to Tachiyomi and downloading will proceed.

Yeah, but Android treats all mobile data connections as metered, even when they aren't, AFAIK (I'd be glad to be corrected). I checked on my end, my unmetered 4G connection is seen as metered, with absolutely no ability to change that - which is why it could be confusing, as a user's definition of metered isn't the same as Android's.

**Comment by kaiteck** [comment]
I don't think "Only download over Wi-Fi" should be renamed but a checkbox asking if the user wants to use the metered connection on the download list should resolve this issue. So if a user enabled "Only download over Wi-Fi", the checkbox will remain unchecked by default.

**Comment by arkon** [comment]
So this was actually changed in #3489 to only consider wifi rather than metered connections.

I'm not sure if metered restrictions are something that users require, but if it is, please open a new issue so I can prioritize it accordingly.
